.travis.yml

```diff
@@ -1,10 +1,12 @@
 language: python
 python:
   - "2.7"
+  - "3.4"
```

Comment:
I don't think you need this if you're using tox env's to control the python versions.